projects/treasury/psyoptions.js

```diff
@@ -0,0 +1,14 @@
+const { treasuryExports } = require("../helper/treasury");
+
+module.exports = treasuryExports({
+  solana: {
+    owners: [
+      'CcNUW7KDCdaUY6rNqYJBmTKYn66RjYTVyPUqCNEiALdp',
+      'CyDnoEMVuf21v23bxoS2wXxPdCvRR2yFLfymegMH1WY4',
+      'E4tuwwYvmB9XWKTmhxeRmywhMmttyDLtMok6jQVAbDLG'
+    ],
+    ownTokens: [
+      '2v6gYajLxSd2xdfRMz8LCUJrc79Z26PTGmVzqLA5g6KW', // PSY
```

Comment:
this is a token account, the token is: PsyFiqqjiv41G7o5SMRzDJCu4psptThNR2GtfeGHfSq

https://solscan.io/token/PsyFiqqjiv41G7o5SMRzDJCu4psptThNR2GtfeGHfSq